src/Patches/PlayerControllerPatch.cs

```diff
@@ -10,87 +12,116 @@ namespace OoLunar.LethalCompanyPatched.Patches
     [HarmonyPatch(typeof(PlayerControllerB)), LethalPatch]
     internal class PlayerControllerPatch
     {
-        private static readonly FieldInfo _playerCarryWeight = typeof(PlayerControllerB).GetField("carryWeight");
-        private static bool _tempCrouch = false;
+        private static readonly FieldInfo _playerCarryWeight = typeof(PlayerControllerB).GetField("carryWeight");            
+        private static readonly FieldInfo? _slipperinessConfigField = AccessTools.Field(typeof(LethalCompanyPatchedPlugin), nameof(LethalCompanyPatchedPlugin.Slipperiness));
```

Comment:
I'm personally not familiar with the `AccessTools` class as I normally would just use plain old reflection + delegates. Seems helpful though

Reply:
If you don't like AccessTools, you don't have to keep it. I just used it because I was learning through harmony example code.